archivers/libdeflate/Portfile

```diff
@@ -32,3 +32,13 @@ checksums           rmd160  81681b7e53369ebf4fdb11e02dcc179140290b45 \
                     size    158379
 
 makefile.has_destdir yes
+
+# Fix install_name of dylib
+pre-destroot {
+    foreach file [glob ${worksrcpath}/*.dylib ${worksrcpath}/*.dylib] {
```

Comment:
Why is the glob listing `${worksrcpath}/*.dylib` twice?

Reply:
Hmm, good question. It was a copy pasta 🍝  from: https://github.com/macports/macports-ports/blob/master/java/hadoop/Portfile#L89-L97.

The second one isn't necessary, I'll get rid of it.

Reply:
Hmm, I removed it, uninstalled, did `sudo port -vst install`, and was back to not having the correct install_name for the dylib, per `otool -L`.

So then I added it back, etc., and still don't have the correct install_name.

I'm not sure what's happening, but I have to go afk for several hours, will investigate further later today. Maybe I'm hitting some kind of caching behavior? But that would be weird because I'm running `sudo port -vst install` in the `archivers/libdeflate` directory where the modified `Portfile` resides.

Reply:
I've force-pushed some changes that work reliably on my machine, i.e. install_name is correct for `libdeflate.0.dylib`:
```
$ cd /opt/local/lib

$ otool -L libdeflate.0.dylib
libdeflate.0.dylib:
	/opt/local/lib/libdeflate.0.dylib (compatibility version 0.0.0, current version 0.0.0)
	/usr/lib/libSystem.B.dylib (compatibility version 1.0.0, current version 1311.0.0)

$ otool -L libdeflate.dylib 
libdeflate.dylib:
	/opt/local/lib/libdeflate.0.dylib (compatibility version 0.0.0, current version 0.0.0)
	/usr/lib/libSystem.B.dylib (compatibility version 1.0.0, current version 1311.0.0)

$ otool -L libnotcurses-core.dylib 
libnotcurses-core.dylib:
	/opt/local/lib/libnotcurses-core.3.dylib (compatibility version 3.0.0, current version 3.0.5)
	/opt/local/lib/libdeflate.0.dylib (compatibility version 0.0.0, current version 0.0.0)
	/opt/local/lib/libncurses.6.dylib (compatibility version 6.0.0, current version 6.0.0)
	/opt/local/lib/libunistring.2.dylib (compatibility version 5.0.0, current version 5.0.0)
	/usr/lib/libSystem.B.dylib (compatibility version 1.0.0, current version 1311.0.0)
```
I think this PR is good to go, though I realize @dankamongmen may want to look into it further.